### PR TITLE
New file copying fix

### PR DESCRIPTION
**Pull request by monkeyman192**
This should fix #64 
I had to slightly revert how the raw_fname was assigned as it was being assigned without a file extension which caused issues when trying to find the actual file. Figured the way I did was better than splitting then giving it an extension again...

### CODE REVIEW

**Comment by jasmainak** [comment]
@monkeyman192 you can use master of bids-validator to make the test pass

**Comment by monkeyman192** [comment]
I don't really understand the issues Travis is having...

**Comment by jasmainak** [comment]
I restarted the build once more ... let's see if it works now

**Comment by agramfort** [comment]
I see the webhook at https://github.com/mne-tools/mne-bids/settings/hooks

so I don't know.

did you add the repo in circle ci site?

**Comment by jasmainak** [comment]
umm ... it's travis ci which is failing

File: mne_bids/mne_bids.py [hunk 1]
```diff
@@ -481,39 +481,25 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
...
-                if ext == '.ds':
-                    # it is actually a folder and we need to copy the
-                    # entire folder
-                    sh.copytree(raw_fname, raw_file_bids)
-                else:
-                    # copy the file
-                    sh.copyfile(raw_fname, raw_file_bids)
+    if ext not in ALLOWED_EXTENSIONS:
+        raise ValueError('ext must be in %s, got %s'
+                         % (''.join(ALLOWED_EXTENSIONS), ext))
```
Comment (jasmainak) [inline mne_bids/mne_bids.py]: @monkeyman192 can you add a test for this? Thanks!

Comment (sappelhoff) [reply mne_bids/mne_bids.py]: looks much better @jasmainak - thanks!

**Comment by jasmainak** [comment]
Thanks @monkeyman192 for the fixes

**Pull request merged**